__tests__/integration/one-app.spec.js

```diff
@@ -85,6 +88,31 @@ describe('Tests that require Docker setup', () => {
       expect(rawHeaders).not.toHaveProperty('access-control-allow-credentials');
     });
 
+    describe('metrics', () => {
+      it('connects', async () => {
+        expect.assertions(1);
+        const response = await fetch(appAtTestUrls.fetchMetricsUrl);
+        expect(response).toHaveProperty('status', 200);
+      });
+
+      it('has all metrics', async () => {
+        expect.assertions(1);
+        const response = await fetch(appAtTestUrls.fetchMetricsUrl);
+        const parsedMetrics = parsePrometheusTextFormat(await response.text());
+        expect(parsedMetrics.map((metric) => metric.name).sort()).toMatchSnapshot();
+      });
+
+      it('has help information on each metric', async () => {
+        expect.assertions(1);
+        const response = await fetch(appAtTestUrls.fetchMetricsUrl);
+        const parsedMetrics = parsePrometheusTextFormat(await response.text());
+        const errors = parsedMetrics
+          .filter((metric) => (!metric.help) || metric.help.length === 0)
+          .map((metric) => metric.name);
+        expect(errors).toEqual([]);
```

Comment:
`it('has help information on each metric', async () => {` asserts `expect(errors).toEqual([]);`. It seems this test should be renamed to something related to `it('has no errors')` instead?

Reply:
After entering @PixnBits brain, he tends to test the negative scenario and if it can't find anything then it is a successful test, the errors keyword is generic there I think 🤔

Reply:
My thought was showing all, say, 20 errors at once would be easier to see what's going on, rather than stopping on the first bad assertion and not trying/reporting the other 19 issues.

Reply:
I see your reasoning. Any chance we can update the assertion text to be more descriptive related to your point? If I were to maintain or update `parsePrometheusTextFormat` I wouldn't be sure what this test case's purpose.

Reply:
as discussed I'll update this to test for the positive scenario 👍